src/app/log-in/log-in.component.ts

```diff
@@ -45,8 +45,6 @@ export class LogInComponent implements LoggedInCallback, OnInit {
     console.log('isLoggedInMethod');
     if (isLoggedIn) {
        this.router.navigate(['/home']);
-    } else {
-       this.router.navigate(['/login']);
```

Comment:
Just curious what will happen if user cannot login in if we remove this else condition?

Reply:
i have updated that! It is now added back in and routes to '/landing' as routing to '/login' would just be a seperate login page again. pushed the newest commit